cumulusci/tasks/metadata_etl/objects.py

```diff
@@ -0,0 +1,65 @@
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.utils import process_list_arg
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+
+OBJ_SETTINGS = [
```

Comment:
I like this design choice a lot.

Reply:
Yeah, I originally had it where you have to pass `enableReports` instead of just `Reports`.  The drawback is that you have to list all the choices in the description for the option because I don't know of any shorter way to even explain what the value should be "take the element name like enableReports and remove the enable" seems confusing ;)